src/output-json-ike.c

```diff
@@ -179,6 +199,4 @@ void JsonIKELogRegister(void)
     OutputRegisterTxSubModule(LOGGER_JSON_IKE, "eve-log", "JsonIKELog", "eve-log.ike",
             OutputIKELogInitSub, ALPROTO_IKE, JsonIKELogger, JsonIKELogThreadInit,
             JsonIKELogThreadDeinit, NULL);
-
-    SCLogDebug("IKE JSON logger registered.");
```

Comment:
Why did you remove this ?

Reply:
Got the impression that newer parsers don't log that, don't know if there is a best practice....

Reply:
Ok for me